docs/resources/hvn.md

```diff
@@ -70,6 +70,10 @@ Optional:
 Import is supported using the following syntax:
 
 ```shell
-# The import ID is {hvn_id}
+# Using an explicit project ID, the import ID is:
+# {project_id}:{hvn_id}
+terraform import hcp_hvn.example f709ec73-55d4-46d8-897d-816ebba28778:main-hvn
+# Using the provider-default project ID, the import ID is:
+# {hvn_id}
```

Comment:
I was avoiding creating resource-specific templates if there are other clear ways of representing data because AFAIK it prevents resources from being automatically adjusted if/when the default template changes. 

I'm not sure if there actually _is_ a scenario where we would actually update the default template. If there isn't one, then there's definitely no harm in creating custom templates here.

Reply:
Alternatively you can do something like the following to reduce the context of the code block

```shell
# {project_id}:{hvn_id}
terraform import hcp_hvn.example f709ec73-55d4-46d8-897d-816ebba28778:main-hvn
# {project_id} not required when setting a provider-d
terraform import hcp_hvn.example main-hvn
```

![image (1)](https://github.com/hashicorp/terraform-provider-hcp/assets/1749304/7a3504fa-c1af-43c4-8d08-95e44d4c68a7)
efault project ID: